homeassistant/components/mill/sensor.py

```diff
@@ -155,3 +190,27 @@ def available(self) -> bool:
     def _update_attr(self, device):
         self._available = device.available
         self._attr_native_value = getattr(device, self.entity_description.key)
+
+
+class LocalMillSensor(CoordinatorEntity, SensorEntity):
+    """Representation of a Mill Sensor device."""
+
+    def __init__(self, coordinator, entity_description):
+        """Initialize the sensor."""
+        super().__init__(coordinator)
+
+        self.entity_description = entity_description
+        self._attr_name = (
+            f"{coordinator.mill_data_connection.name} {entity_description.name}"
+        )
+        self._update_attr()
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle updated data from the coordinator."""
+        self._update_attr()
+        self.async_write_ha_state()
+
+    @callback
+    def _update_attr(self) -> None:
```

Comment:
For this coordinator entity it's more efficient to implement the `native_value` property. Then we don't need the `_update_attr` method and we don't need to implement `_handle_coordinator_update`.